### ISSUE TITLE
Provide the partitionKey as part of the Event

### ISSUE THREAD

**Issue by aquamatthias** [WW-Digital/reactive-kinesis#32]
Hey,

we are currently playing with reactive-kinesis. As first impression: It looks really good - so thanks for this cool library. 
What I miss is the information about the partitionKey in each event which is available in `Record`.
If a shard is taken over by a different worker, we would like to drop buffered events from this shard.
Do you think it would make sense to add the key to the `ConsumerEvent` class?
Happy to create a PR, if you agree.

**Comment by markglh** [comment]
Thanks @aquamatthias - apologies for the delay I got accidentally removed as a contributor so didn't receive the notifications. 

This makes sense to me! I didn't think of that use case, you'd generally get notified as the worker would shutdown if I understand correctly? But additional information is all good!

**Comment by aquamatthias** [comment]
@markglh Exactly. You can see one use case that I had in mind in #33 in the Akka Stream Source.
If a lease is lost I would like to throw away all buffered messages of this shard.
